Atomic/util.py

```diff
@@ -447,7 +445,7 @@ def default_docker():
 def default_docker_lib():
     try:
         return get_docker_conf()["DockerRootDir"]
-    except ValueError:
+    except (NoDockerDaemon, requests.ConnectionError):
```

Comment:
Any value to adding a warning here with verbiage similar to the error at the old line 60?